desktop/test/run-visdiff.js

```diff
@@ -15,12 +16,36 @@ const DIFF_REMOVED = 'removed'
 const DIFF_CHANGED = 'changed'
 const DIFF_SAME = 'same'
 
+const DRY_RUN = !!process.env['VISDIFF_DRY_RUN']
+
+function packageHash () {
+  return crypto.createHash('sha1').update(fs.readFileSync('package.json')).digest('hex')
+}
+
+function checkout (commit) {
+  const origPackageHash = packageHash()
+  execSync(`rm -rf node_modules.${origPackageHash} && mv node_modules node_modules.${origPackageHash}`)
```

Comment:
This line moves the current `node_modules` out of the way while retaining it in case it can be reused by the other commit. The `rm` is because an existing `node_modules` with the same hash is assumed to have the same contents.